libr/core/cmd_debug.inc.c

```diff
@@ -1393,6 +1393,48 @@ static void cmd_debug_modules(RCore *core, int mode) { // "dmm"
 	r_list_free (list);
 }
 
+R_API double get_glibc_version(RCore *core, const char *libc_path) {
+	double version = 0.0f;
+	RBin *bin = r_bin_new ();
+	RIO *io = r_io_new ();
+	r_io_bind (io, &bin->iob);
+	RBinFileOptions opt = { 0 };
+
+	if (r_bin_open (bin, libc_path, &opt)) {
```

Comment:
maybe worth adding some extra checks about the filetype. but guess its fine for an intiaial approach.

Reply:
Checks such as what? Like making sure it's an .so?

Reply:
Not sure if its worth ensuring its an elf too. But i doubt this function will take a wrong path as arg